packages/relic/lib/src/headers/typed/typed_headers.dart

```diff
@@ -38,3 +37,11 @@ export 'package:relic/src/headers/typed/headers/access_control_allow_origin_head
 export 'package:relic/src/headers/typed/headers/access_control_allow_headers_header.dart';
 export 'package:relic/src/headers/typed/headers/access_control_expose_headers_header.dart';
 export 'package:relic/src/headers/typed/headers/accept_header.dart';
+
+/// A typed header that can be converted to a header string.
+abstract class TypedHeader {
```

Comment:
Suggestion: Mark this as an `abstract interface class` and implement it instead.

Since inheritance is not really needed for the usecase, it should be enough to guarantee that a method is implemented right?

Reply:
Yes.